elks/arch/i86/drivers/char/kbd-bios.c

```diff
@@ -81,6 +81,5 @@ static void restart_timer(void)
 
 void kbd_init(void)
 {
-    enable_irq(1);		/* enable BIOS Keyboard interrupts */
```

Comment:
I suppose it is things like this that caused you to write this PR, the idea being that BIOS had previous functionality and ELKS should leave it alone, right?

Since this is in polling BIOS driver, I suppose leave change, since easy workaround with configuration. This could be used as test on real hardware to determine how accurate our thoughts are on BIOS interrupts. (Do you have any other outside information on BIOS's using interrupts??)

Reply:
You suppose right. IRQ1 is already activated, otherwise we could not use INT 16h functions.

Reply:
Such supposition is not proven, there are old BIOS that have bugs. IRQ 1 is not required by hardware for reading keyboard, and ELKS only uses INT 16h polling functions. But agree we can test on this small case easily on real hardware, at least on newer systems.